example/generator/WorkerInterface.js

```diff
@@ -72,14 +72,14 @@ define([
         });
         var messageId;
 
-        function callback(message) {
+        let callback = (message => {
```

Comment:
Why is the callback enclosed in brackets? The correct way to write this would be (message) => {}

Reply:
This was incorrect, but what I meant to do was the format:
`let name = argument => { //do something };`

According to https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions brackets are optional for arrow functions with a single argument. I don't see anything in our standards saying otherwise, so I just went with what looks cleaner.